s3watcher/lib/Transfer.js

```diff
@@ -37,6 +37,23 @@ module.exports = {
             console.log(stateMessage, state);
         }
 
+       const success = function(result) {
+            log("delete", s3Object);
+
+            return S3Helper.deleteS3Object(s3Object);
+        };
+
+        const failGraceful = function(e) {
+            log("failed", e)
+
+            log("copy", s3CopyObject);
+            return S3Helper.copyS3Object(s3CopyObject).flatMap(function(){
```

Comment:
Had to double check, but it seems not: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html